openbb_terminal/stocks/screener/finviz_model.py

```diff
@@ -188,7 +188,9 @@ def get_screener_data(
             else:
                 df_screen = screen.screener_view(ascend=ascend)
 
-    df_screen = df_screen.rename(columns={"\n\nTicker": "Ticker"})
+    df_screen.rename(
+        columns={"\n\nTicker": "Ticker", "Ticker\n\n": "Ticker"}, inplace=True
```

Comment:
inplace bad.  but ill allow it

Reply:
We're you saving those bad headers for downstream? :P

Reply:
lol.  not sure where joke line is, but should be df_screen = df_screen.rename(...

Using inplace is frowned on:
https://stackoverflow.com/questions/45570984/in-pandas-is-inplace-true-considered-harmful-or-not